README.md

```
@@ -1,2 +1,51 @@
+---
+
+## ▶️ Running Locally
+
+1. Start the server at the root of the repo:
+   ```bash
+   python -m viewmaster.viewmaster
+   ```
+
+2. Open your browser and go to:
+
+   **http://127.0.0.1:8080/**
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)